Building teleport via docker locally fails

### Description

**What happened**:
first step on docker readme:
```
$ make docker
$ cd docker
$ make build
docker build -t teleport:latest .
Sending build context to Docker daemon   55.3kB
Step 1/11 : FROM teleport-buildbox:go1.14.4
pull access denied for teleport-buildbox, repository does not exist or may require 'docker login': denied: requested access to the resource is denied
make: *** [Makefile:85: build] Error 1
```

**What you expected to happen**:
build teleport docker files

**How to reproduce it (as minimally and precisely as possible)**:
delete existing teleport-buildbox docker image locally and then run the above three commands in the teleport directory

### Environment
- OS: Ubuntu 20.04.1 LTS

- Where are you running Teleport?: Locally

--- a/docker/Dockerfile
+++ b/docker/Dockerfile
@@ -1,6 +1,6 @@
 # The base image (buildbox:latest) is built by running `make -C build.assets`
 # from the base repo directory $GOPATH/gravitational.com/teleport
-FROM teleport-buildbox:go1.14.4
+FROM quay.io/gravitational/teleport-buildbox:go1.14.4
 RUN apt-get update
 # DEBUG=1 is needed for the Web UI to be loaded from static assets instead
 # of the binary

